src/App.js

```diff
@@ -1,25 +1,47 @@
-import logo from './logo.svg';
 import './App.css';
+import foodsFromJson from './foods.json';
+import { useState } from "react";
+import AddFoodForm from './components/AddFoodForm';
+import SearchBar from './components/SearchBar';
+import { Divider, Button } from 'antd';
+import FoodList from './components/FoodList';
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+  const [foods, setFoods] = useState(foodsFromJson);
+  const [searchString, setSearchString] = useState('');
+  const [displayAddFoodForm, setDisplayAddFoodForm] = useState(true);
+
+  const addNewFoodItem = (newFoodItem) => {
+    setFoods(oldFoods => [...oldFoods, newFoodItem]);
+  }
+
+  const searchStringChange = (event) => {
+    setSearchString(event.target.value);
+  };
+
+  const deleteFood = foodName => {
+    const filteredFoods = [...foods].filter(food => {
+        return food.name !== foodName;
+    });
+    setFoods(filteredFoods);
+  };
+
+  const handleToggle = () => {
+    return setDisplayAddFoodForm(!displayAddFoodForm);
+  };
+
+  return <div className="App">
+    <Divider>Add food entry</Divider>
+    <div className={displayAddFoodForm ? "hidden" : null}>
```

Comment:
instead of toggling this wrapper class you could also use conditional rendering for this!